src/components/AssetSearch/stories/handleSearchResults.md

```diff
@@ -9,7 +9,7 @@ import 'antd/dist/antd.css';
 
 import React from 'react';
 import { AssetSearch } from '@cognite/gearbox';
-import { Asset } from '@cognite/sdk';
+import { Asset } from '@cognite/sdk/dist/src/types/types';
```

Comment:
In this PR we exposed the types in a much better way than before: https://github.com/cognitedata/cognitesdk-js/pull/213